src/rpc/rawtransaction.cpp

```diff
@@ -35,6 +35,11 @@
 
 using namespace std;
 
+#ifdef ENABLE_WALLET
+std::string HelpRequiringPassphrase(CWallet *);
```

Comment:
I'd think we can (and, more importantly, would) still nag people submitting PRs which add more uses of it outside of src/wallet/rpc*.